src/plugins/wizard/public/application/components/workspace.scss

```diff
@@ -9,4 +10,27 @@
   &__empty {
     height: 100%;
   }
+
+  &__container {
+    position: relative;
+  }
+
+  &__handFieldSvg {
+    animation-name: dragAndDropAnimation;
+    animation-direction: alternate-reverse;
+    animation-duration: 1.5s;
+    animation-iteration-count: infinite;
+    position: absolute;
+    top: 43%;
+  }
+}
+
+@keyframes dragAndDropAnimation {
+  0% {
+    transform: translate3d(65%, 0, 0);
```

Comment:
Why are you using a 3d transformation?

Reply:
translate3d tends to be smoother/a little more performant on some animations because it gets GPU accelerated, especially on mobile